lib/awesome_bot/cli.rb

```diff
@@ -26,8 +26,9 @@ def cli()
         opts.on('--base-url [base url]',           String,    'Base URL to use for relative links') { |val| options['base_url'] = val }
         opts.on('-d', '--request-delay [seconds]', Integer,   'Set request delay')                  { |val| options['delay'] = val }
         opts.on('-t', '--set-timeout [seconds]',   Integer,   'Set connection timeout')             { |val| options['timeout'] = val }
-        opts.on('--skip-save-results',             TrueClass, 'Skip saving results')                { |val| options['no_results'] = val }        
+        opts.on('--skip-save-results',             TrueClass, 'Skip saving results')                { |val| options['no_results'] = val }
         opts.on('-w', '--white-list [urls]',       Array,     'Comma separated URLs to white list') { |val| options['white_list'] = val }
+        opts.on('-v', '--version',                 String,    'Display version')                    { |val| puts "Awesome_bot version #{VERSION}" }
```

Comment:
should be: `puts #{PROJECT} version #{VERSION}`

Reply:
Done @dkhamsing , thanks. Didn't notice that :smile: